.travis.yml

```diff
@@ -2,34 +2,34 @@
 sudo: false
 language: python
 env:
-  - TOXENV=py27-django18-sqlite
-  - TOXENV=py27-django18-mysql
-  - TOXENV=py27-django18-postgres
+  - TOXENV=py27-django18-sqlite PYTHONPATH=/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
+  - TOXENV=py27-django18-mysql PYTHONPATH=/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
+  - TOXENV=py27-django18-postgres PYTHONPATH=/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
   # Meta
-  - TOXENV=sdist
-  - TOXENV=eslint
-  - TOXENV=docs
-  - TOXENV=mo
-  - TOXENV=isort
-  - TOXENV=pep8_flake8
-  - TOXENV=pep257
+  - TOXENV=project PYTHONPATH=/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages  TOX_TESTENV_PASSENV="PYTHONPATH"
 matrix:
   allow_failures:
     - env: TOXENV=isort
+before_cache:
+  - rm -f $HOME/.cache/pip/log/debug.log
 cache:
   directories:
     - pootle/static/js/node_modules
+    - pootle/assets
     - $HOME/.cache/pip
-    - docs/_build
-    - .tox
 before_install:
   # Error as early as possible if we can't compile
   - python -m compileall -q -f -x ".*[.]tox.*" .
 install:
   - pip install --upgrade "pip>=8.0" # FIXME remove once Travis support pip8 by default
   - pip install --upgrade "tox>=2.3"
+  - pip install
+      -r requirements/base.txt
+      -r requirements/tests.txt
+      -e .
+
 script:
-  - tox -e $TOXENV
+  - tox --sitepackages -e $TOXENV
```

Comment:
not sure if we need sitepackages as its set in tox.ini - the problem was with the site-packages folder - so this _might_ be unnessary